app/src/main/java/framgia/com/ichat/screen/home/PagerAdapterHome.java

```diff
@@ -0,0 +1,43 @@
+package framgia.com.ichat.screen.home;
+
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentManager;
+import android.support.v4.app.FragmentPagerAdapter;
+
+import framgia.com.ichat.screen.privateroom.FragmentPrivateRoom;
+import framgia.com.ichat.screen.useronline.FragmentUserOnline;
+
+public class PagerAdapterHome extends FragmentPagerAdapter {
+    private static final int FRAGMENT_PRIVATE_ROOM = 0;
+    private static final int FRAGMENT_USER_ONLINE = 1;
+    private String mTabTitles[];
+
+    public PagerAdapterHome(FragmentManager fm, String[] tabTitles) {
+        super(fm);
+        this.mTabTitles = tabTitles;
+    }
+
+    @Override
+    public Fragment getItem(int i) {
+        switch (i) {
+            case FRAGMENT_PRIVATE_ROOM:
+                return new FragmentPrivateRoom();
```

Comment:
e có thể tạo khởi tạo cho fragment bằng cáhc như sau
https://medium.com/@azjkjensen/using-the-newinstance-pattern-in-kotlin-e40c1b4ba1ef